web/src/features/Dashboard/components/DashboardLicenseCard.tsx

```diff
@@ -25,6 +25,7 @@ type Props = {
   gettingAppLicenseErrMsg: string | null;
   syncCallback: () => void;
   children: React.ReactNode;
+  refetchLicense: () => void;
```

Comment:
The `DashboardLicenseCard` has a local state and a props state for the license. 

It initially renders the props.license. If you trigger a `PUT` fetch by syncing the license, then the app renders the response from that PUT. 

I added a callback to trigger a refetch on the license after the PUT is finished. Not refactoring this state vs props thing yet because it's kind of complex but adding this will ensure that this component is consistently rendering the props.license and that prop is always up to date.

Reply:
why is the refetch on the license necessary after PUT is finished?